internal/cli/atlas/deployments/setup.go

```diff
@@ -187,30 +191,62 @@ func (opts *SetupOpts) configureMongod(ctx context.Context, keyFileContents stri
 		return err
 	}
 
-	// init ReplicaSet
-	if err := opts.waitConnection(opts.Port); err != nil {
+	return opts.initReplicaSet(ctx)
+}
+
+func (opts *SetupOpts) initReplicaSet(ctx context.Context) error {
+	// connect to local deployment
+	const waitDuration = 60 * time.Second
+	ctxConnect, cancel := context.WithTimeout(ctx, waitDuration)
+	defer cancel()
+
+	client, errConnect := mongo.Connect(ctxConnect, mongoOpts.Client().ApplyURI(fmt.Sprintf("mongodb://localhost:%d/?directConnection=true", opts.Port)))
```

Comment:
for testability, I'll create (as part of the current task) a separate PR to wrap the mongo client and RunCommand and InsertOne functions